web/app/controllers/home_controller.rb

```diff
@@ -9,8 +9,12 @@ class HomeController < ApplicationController
   PROD_INDEX_PATH = Rails.public_path.join("dist")
 
   def index
-    contents = File.read(File.join(Rails.env.production? ? PROD_INDEX_PATH : DEV_INDEX_PATH, "index.html"))
+    if ShopifyAPI::Context.embedded? && !params[:embedded].present? || params[:embedded] != "1"
```

Comment:
I don't fully get the conditions here but I think there should be some brackets:
`ShopifyAPI::Context.embedded? && (!params[:embedded].present? || params[:embedded] != "1")`

Because `false && false || true` returns `true` while here I think we want it to return `false`.

Reply:
Thank you ... fixed in the latest version on `main`.